client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -1,4 +1,15 @@
+#if (MATERIAL_TYPE == TILE_MATERIAL_WAVING_LIQUID_TRANSPARENT || \
+	MATERIAL_TYPE == TILE_MATERIAL_WAVING_LIQUID_OPAQUE || \
+	MATERIAL_TYPE == TILE_MATERIAL_WAVING_LIQUID_BASIC || \
```

Comment:
we found out that line-continuations don't work on some drivers, there's a comment somewhere
so this will have to go all on one line